admin/make-release.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+#
+# Create a release from `ref`.
```

Comment:
git uses "commitish" to mean something that maps to a commit - could be a hash, or a tag, or a branch name, or probably we want to be explicit that a tag is expected?

Reply:
I went back to "tag".